components/LoginForm/LoginForm.tsx

```diff
@@ -6,44 +6,34 @@ import getConfig from "next/config";
 
 export function LoginForm(): JSX.Element {
     const {publicRuntimeConfig} = getConfig();
-    const [userId, setUserId] = useState("");
-    const [password, setPassword] = useState("");
+    const [userEmail, setUserEmail] = useState("");
+    const [userPassword, setUserPassword] = useState("");
     const router = useRouter();
 
     function credentialsAreValid(statusCode: number): boolean {
-
-        if (statusCode == 403) {
-            alert("Those details are not in our system! Please try again");
-            return false;
-        }
-        if (statusCode != 200) {
+        if (statusCode == 200) {
+            return true;
+        } else if (statusCode == 403) {
+            alert("Incorrect login details")
+            return false
+        } else {
             alert("Something went wrong, please try again.");
             return false;
         }
```

Comment:
have you seen `switch` statements?
In some cases like this they can give some really clear and tidy syntax! 😊

```ts
switch (statusCode) {
    case 200:
        return true;
    case 403:
        alert("Incorrect login details");
        return false;
    default:
        alert("Something went wrong, please try again");
        return false;
}
```

Reply:
I do know switch and that is a very good idea!

Reply:
Converted to switch statement now